test/flags.js

```diff
@@ -0,0 +1,242 @@
+});
+
+metatests.test('FlagsClass.toString', test => {
+  const Numeri = Flags.from('Uno', 'Due', 'Tre', 'Quatro');
+  const nums = [
+    [[], '0000'],
+    [['Uno'], '0001'],
+    [['Due'], '0010'],
+    [['Tre'], '0100'],
+    [['Quatro'], '1000'],
+    [['Uno', 'Due'], '0011'],
+    [['Due', 'Tre'], '0110'],
+    [['Uno', 'Quatro'], '1001'],
+    [['Tre', 'Quatro'], '1100'],
+    [['Uno', 'Due', 'Tre', 'Quatro'], '1111'],
+  ];
+  nums.forEach(num =>
+    test.strictSame(Numeri.from(...num[0]).toString(), num[1]));
+
+  test.end();
+});
+
+metatests.test('FlagsClass.toNumber', test => {
+  const Numeri = Flags.from('Uno', 'Due', 'Tre', 'Quatro');
+  const nums = [
+    [[], false],
+    [['Uno'], 1],
+    [['Due'], 2],
+    [['Tre'], 4],
+    [['Quatro'], 8],
+    [['Uno', 'Due'], 3],
+    [['Due', 'Tre'], 6],
+    [['Uno', 'Quatro'], 9],
+    [['Tre', 'Quatro'], 12],
+    [['Uno', 'Due', 'Tre', 'Quatro'], 15],
+  ];
+  nums.forEach(num =>
+    test.strictSame(Numeri.from(...num[0]).toNumber(), new Uint64(num[1])));
+
+  test.end();
+});
+
+metatests.test('FlagsClass.toPrimitive', test => {
+  const Numeri = Flags.from('Uno', 'Due', 'Tre', 'Quatro');
+  const nums = [
+    [[], 0],
+    [['Uno'], 1],
+    [['Due'], 2],
+    [['Tre'], 4],
+    [['Quatro'], 8],
+    [['Uno', 'Due'], 3],
+    [['Due', 'Tre'], 6],
+    [['Uno', 'Quatro'], 9],
+    [['Tre', 'Quatro'], 12],
+    [['Uno', 'Due', 'Tre', 'Quatro'], 15],
+  ];
+  nums.forEach(num =>
+    test.strictSame(+Numeri.from(...num[0]), num[1]));
+
+  test.end();
+});
+
+metatests.test('Create new instance with too much arguments', test => {
+  const values = new Array(65);
+  const message = 'Flags does not support more than 64 values';
+  test.throws(() => {
+    Flags.from(...values);
+  }, new Error(message));
+  test.end();
+});
+
+metatests.test('Get incorrect key', test => {
+  const Numeri = Flags.from('Uno', 'Due', 'Tre', 'Quatro');
+  const num = Numeri.from();
+  const message = 'Flags instance does not have key Cinque';
+  test.throws(() => {
+    num.get('Cinque');
+  }, new Error(message));
+  test.end();
+});
+
+metatests.test('Set incorrect key', test => {
+  const Numeri = Flags.from('Uno', 'Due', 'Tre', 'Quatro');
+  const num = Numeri.from();
+  const message = 'Flags instance does not have key Cinque';
+  test.throws(() => {
+    num.set('Cinque');
+  }, new Error(message));
+  test.end();
+});
+
+metatests.test('Unset incorrect key', test => {
+  const Numeri = Flags.from('Uno', 'Due', 'Tre', 'Quatro');
+  const num = Numeri.from();
+  const message = 'Flags instance does not have key Cinque';
+  test.throws(() => {
+    num.unset('Cinque');
+  }, new Error(message));
+  test.end();
+});
+
+metatests.test('Toggle incorrect key', test => {
+  const Numeri = Flags.from('Uno', 'Due', 'Tre', 'Quatro');
+  const num = Numeri.from();
+  const message = 'Flags instance does not have key Cinque';
+  test.throws(() => {
+    num.toggle('Cinque');
+  }, new Error(message));
```

Comment:
We should probably test for `TypeError` here:
```suggestion
  }, new TypeError(message));
```